stockflux-launcher/public/windowHandler.js

```diff
@@ -0,0 +1,4 @@
+const onChange = searchInput => {
+  // eslint-disable-next-line no-undef
+  fin.desktop.InterApplicationBus.publish('search-request', searchInput);
```

Comment:
We should avoid using the legacy desktop namespace where possible (fin.InterApplicationBus)

Reply:
Agreed, thanks for noticing. Fixed -> c5bba2ac17e81d4069a1c54bc0a431f676deb99b